src/main/java/uk/ac/ebi/ena/webin/cli/context/sequence/SequenceManifestReader.java

```diff
@@ -67,7 +72,9 @@ public SequenceManifestReader(
                     .file().optional().name( Field.TAB          ).desc( Description.TAB          ).processor( getTabProcessors() ).and()
                     .file().optional().name( Field.FLATFILE     ).desc( Description.FLATFILE     ).processor( getFlatfileProcessors() ).and()
                     .meta().optional().name( Field.AUTHORS      ).desc( Description.AUTHORS      ).processor(new AuthorProcessor()).and()
-                    .meta().optional().name( Field.ADDRESS      ).desc( Description.ADDRESS      )
+                    .meta().optional().name( Field.ADDRESS      ).desc( Description.ADDRESS      ).and()
+                    .meta().optional().name(ReadsManifestReader.Field.SUBMISSION_TOOL).desc(ReadsManifestReader.Description.SUBMISSION_TOOL).and()
+                    .meta().optional().name(ReadsManifestReader.Field.SUBMISSION_TOOL_VERSION).desc(ReadsManifestReader.Description.SUBMISSION_TOOL_VERSION)
```

Comment:
Should not point to ReadsManifestReader Field/Description. Should use the fields from this class. Or we could add them into a common class.